src/cpp/enclave/CMakeLists.txt

```diff
@@ -33,10 +31,38 @@ if ("$ENV{DEBUG}" EQUAL 1)
   target_compile_definitions(enclave_trusted PUBLIC -DDEBUG)
 endif()
 
-add_compile_options(${COMMON_COMPILE_FLAGS} -fvisibility=hidden -fpie -fstack-protector -Wl,--no-undefined -nostdlib -nodefaultlibs -nostartfiles -Wl,-Bstatic -Wl,-Bsymbolic -Wl,--no-undefined -Wl,-pie -Wl,--export-dynamic -Wl,--defsym,__ImageBase=0)
+add_compile_options(${COMMON_COMPILE_FLAGS} -fvisibility=hidden -fpie -fstack-protector -Wno-unused-parameter -Wno-unused-variable -Wl,--no-undefined -nostdlib -nodefaultlibs -nostartfiles -Wl,-Bstatic -Wl,-Bsymbolic -Wl,--no-undefined -Wl,-pie -Wl,--export-dynamic -Wl,--defsym,__ImageBase=0)
 target_compile_definitions(enclave_trusted PUBLIC OE_API_VERSION=2)
 set_target_properties(enclave_trusted PROPERTIES POSITION_INDEPENDENT_CODE OFF)
 
+########## Fetch and make opaque-utils library available ##########
+include(FetchContent)
+FetchContent_Declare(
+  mc2_utils_e
+  GIT_REPOSITORY https://github.com/mc2-project/mc2-utils.git
+)
+set(FETCHCONTENT_QUIET OFF)
+set(HOST OFF CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(mc2_utils_e)
+add_compile_definitions(SPDLOG_NO_THREAD_ID FMT_USE_INT128=0)
+
+include_directories(${spdlog_SOURCE_DIR}/include)
```

Comment:
It is: https://github.com/octaviansima/opaque/blob/472375cc16a9095bc4ad9366d164ee0a53b38aa1/src/cpp/app/CMakeLists.txt#L13

Just not including the header yet since we don't actually call it anywhere.